pandas/types/cast.py

```diff
@@ -861,3 +861,9 @@ def _possibly_cast_to_datetime(value, dtype, errors='raise'):
             value = _possibly_infer_to_datetimelike(value)
 
     return value
+
+
```

Comment:
IOW, just run it thru the standard types for now, and asserting that it raises for pandas dtypes